pkg/apiserver/api_installer.go

```diff
@@ -517,6 +531,10 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			namespaced = ""
 		}
 
+		if kubeVerb, found := toDiscoveryKubeVerb[action.Verb]; found {
```

Comment:
if we don't find a mapping here, I think that's an error we'd want to fix, not ignore